.drone.yml

```diff
@@ -47,10 +44,6 @@ steps:
     username:
       from_secret: docker_username
   when:
-    instance:
```

Comment:
why removing this part? The other drone doesn't have the credential for pushing the image.

Reply:
I was thinking we only have `drone-publish.longhorn.io` instance. What are the other instances we have?

Reply:
I saw that `drone-pr.longhorn.io` instance